### PR TITLE
Adding Soli to test cases

### PR DESCRIPTION
**Pull request by JakobWegmann**
### What problem do you want to solve?

Adding Soli to the lohnsteuer test cases in GETTSIM.

### CODE REVIEW

**Review by MImmesberger — COMMENTED** [review]
Thanks, looks good!
The only problem that I noticed is that we now have a lot of code that is copied&pasted (the csv to yaml part). I would suggest to create a separate module that contains the functions that are needed for both the Lohnsteuer and the data on retirement ages. The conversion scripts then call functions from the new module.

**Comment by JakobWegmann** [comment]
My understanding is that the retirement age script is no longer needed as there will be no new test data in the future that comes as a csv file. So perhaps we could just delete it?

**Comment by hmgaudecker** [comment]
Just leave it there, won't hurt here. Thanks!

File: test_data_conversion_scripts/instructions_lohnsteuer.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Converting test data in xlsx from BMF to the YAML file in gettsim tests
+
+## Source of the test data
+The test data is sent out on a mailing list by Referat IA6 together with the new Programmablaufplan. To be added to the mailing list, best contact Steuerrechner@bmf.bund.de
+
+## Instructions for new test data
+1. Save test data for year `year` in `\gettsim-code-for-picking\original_testfaelle` with the name `lohnsteuer_bmf_`year`.xlsx`.
```
Comment (hmgaudecker) [inline test_data_conversion_scripts/instructions_lohnsteuer.md:7]:
```suggestion
1. Save test data for year 20AB in `gettsim-code-for-picking/original_testfaelle` with the name `lohnsteuer_bmf_20AB.xlsx`.
```

File: test_data_conversion_scripts/instructions_lohnsteuer.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+2. In `convert_lohnst.py` replace the line `lst_data["year"] = `year`` with the correct year.
```
Comment (hmgaudecker) [inline test_data_conversion_scripts/instructions_lohnsteuer.md:8]:
```suggestion
2. In `convert_xlsx_tests_csv_lohnst.py` replace the line `lst_data["year"] = 20CD` with the correct year 20AB.
```

File: test_data_conversion_scripts/instructions_lohnsteuer.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+3. Run `convert_lohnst.py`. This generates the file `lohnst_converted.csv`.
+4. Run `convert_cvs_tests_to_yamk.py`. This generated the test files in _gettsim_tests in the gettsim repo.
```
Comment (hmgaudecker) [inline test_data_conversion_scripts/instructions_lohnsteuer.md:10]:
```suggestion
4. Run `convert_cvs_tests_to_yaml.py`. This generated the test files in _gettsim_tests in the gettsim repo.
```
Probably need to say something about the folder structure here? Or do you look for the installation location of GETTSIM based on the import?

File: test_data_conversion_scripts/instructions_lohnsteuer.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Converting test data in xlsx from BMF to the YAML file in gettsim tests
+
+## Source of the test data
+The test data is sent out on a mailing list by Referat IA6 together with the new Programmablaufplan. To be added to the mailing list, best contact Steuerrechner@bmf.bund.de
+
+## Instructions for new test data
+1. Save test data for year `year` in `\gettsim-code-for-picking\original_testfaelle` with the name `lohnsteuer_bmf_`year`.xlsx`.
+2. In `convert_lohnst.py` replace the line `lst_data["year"] = `year`` with the correct year.
+3. Run `convert_lohnst.py`. This generates the file `lohnst_converted.csv`.
```
Comment (hmgaudecker) [inline test_data_conversion_scripts/instructions_lohnsteuer.md:9]:
```suggestion
3. Run `convert_xlsx_tests_csv_lohnst.py`. This generates the file `lohnst_converted.csv`.
```

**Review by hmgaudecker — APPROVED** [review]
Excellent, thanks! All sounds good, also in the sense that in the future, others will be able to carry the torch.

**Pull request merged**